functions-and-scope.js

```diff
@@ -52,14 +78,39 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade(grades) geeft 6.642857142857143
 // averageGrade([6, 4, 5]) geeft 5
 // averageGrade([8, 9, 4, 6, 10]) geeft 7.4
-
+console.log(averageGrade(grades));
+console.log(averageGrade([6, 4, 5]));
+console.log(averageGrade([8, 9, 4, 6, 10]));
+split();
+
+function averageGrade(grades) {
+    let gradesTotal = 0;
+    for (let i = 0; i < grades.length; i++) {
+        gradesTotal += grades[i];
+        if (i === grades.length - 1) {
+            return gradesTotal / (i + 1);
```

Comment:
Je had ook verder kunnen gaan na de for-loop. Nu schrijf je onnodig een extra if statement die wacht tot de for-loop is afgelopen.